app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    args = command.split()
+
+    if len(args) != 3 or args[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv <source> <destination>"
+        )
+
+    source, destination = args[1], args[2]
+
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"Source file '{source}' does not exist.")
+
+    if destination.endswith("/"):
```

Comment:
Consider adding a check to ensure that the destination is not the same as the source. Moving a file to itself should raise an error to prevent unnecessary operations.